Composer/packages/client/src/recoilModel/dispatchers/setting.ts

```diff
@@ -89,12 +89,32 @@ export const settingsDispatcher = () => {
     }
   );
 
+  const updateSkillsInSetting = useRecoilCallback(
+    ({ set, snapshot }: CallbackInterface) => async (skillName: string, skillInfo: Partial<Skill>) => {
+      const currentSettings: DialogSetting = await snapshot.getPromise(settingsState);
+      const matchedSkill = get(currentSettings, `skill[${skillName}]`, undefined);
```

Comment:
Isn't `get`'s third parameter `undefined` by default?
```suggestion
      const matchedSkill = get(currentSettings, `skill[${skillName}]`);
```

Reply:
It is. But just wanted to be more explicit.